Makefile

```diff
@@ -126,5 +146,3 @@ clean:
 
 .PHONY: all clean
 
-# Don't echo commands unless the caller exports "V=1"
-${V}.SILENT:
```

Comment:
One last thing, we added this in to clean up the compilation output. Was introduced here https://github.com/elixir-sqlite/exqlite/pull/181

Reply:
oops sorry about that one.. just a sec